src/scripts/sync_synonym.py

```diff
@@ -0,0 +1,225 @@
+"""Create outputs for syncing synonyms between Mondo and its sources."""
+import os
+import sys
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Union, List, Tuple, Set, Dict
+
+import pandas as pd
+from oaklib import get_adapter
+from oaklib.implementations import SqlImplementation
+from oaklib.types import CURIE, PRED_CURIE
+
+HERE = Path(os.path.abspath(os.path.dirname(__file__)))
+SRC_DIR = HERE.parent
+PROJECT_ROOT = SRC_DIR.parent
+sys.path.insert(0, str(PROJECT_ROOT))
+from src.scripts.utils import PREFIX_MAP, get_owned_prefix_map
+
+
+NON_SOURCE_ID_SOURCE_SPECIFIC_SYNONYM_XREFS = ['OMIM:genemap2']  # current as of 2027/07/15
```

Comment:
## Filtering of `NON_SOURCE_ID_SOURCE_SPECIFIC_SYNONYM_XREFS`
@twhetzel Handled this. Thanks for the analysis!

Reply:
@twhetzel I found another case: `DOID:RELATED`